src/crypto/CHIPCryptoPALmbedTLS.cpp

```diff
@@ -278,8 +278,12 @@ CHIP_ERROR HKDF_SHA256(const uint8_t * secret, const size_t secret_length, const
     return error;
 }
 
-CHIP_ERROR pbkdf2_sha256(const uint8_t * password, size_t plen, const uint8_t * salt, size_t slen, unsigned int iteration_count,
-                         uint32_t key_length, uint8_t * output)
+PBKDF2_sha256::PBKDF2_sha256(void) {}
```

Comment:
Should we move the implementation of empty constructor and destructor to the header file. Currently, it's there in all three impl files.

Reply:
Sure. Did as suggested